### DISCUSSION TITLE
Error in site after deploying on vercel: Application error: a server-side exception has occurred (see the server logs for more information).

### DISCUSSION THREAD

**Discussion by nicitaacom** [nextauthjs/next-auth#9138] [Help]
### Reproduction URL

https://github.com/nicitaacom/acc2-auth_form_next_supabase_2

### Describe the issue

I got this issue on my site after deploying
```
Application error: a server-side exception has occurred (see the server logs for more information).
Digest: 3984128183
```
![image-with-error](https://i.imgur.com/lbtIQGF.png)

Live demo - https://acc2-auth-form-next-supabase-2-9h5afmpbu-nicitaacom.vercel.app/

### How to reproduce

1. Fork repository
2. Try to deploy this on vercel
3. Get this error

### Expected behavior

I wait that it will work but not it throws error in production

## Additional info
package.json
```json
    "next": "^14.0.2",
    "next-auth": "^4.24.5",
```

**Comment by khaledebdelli** [comment]
same issue with ` "next": "13.5",
    "next-auth": "^4.20.1",`

**Reply by nicitaacom** [reply]
Please vote up this github bugs

![image](https://github.com/nextauthjs/next-auth/assets/39565703/8192039b-4ebb-456e-bcf7-fa1791dcb3e5)

https://github.com/orgs/community/discussions/76410

**Comment by kralion** [comment]
same here with t3 stack 


<img width="1073" alt="image" src="https://github.com/nextauthjs/next-auth/assets/66649036/2f65dc08-300a-4482-9b36-42d43945cb46">

**Reply by 0xsaad76** [reply]
did you find a solution for this problem yet ?

**Reply by kralion** [reply]
Nah man, i deleted the projects and migrate to new versions

**Comment by mateBe95** [comment]
same for me
